linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_enable_iommu_force/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+title: 'IOMMU configuration directive'
+
+description: |-
+    On x86 architecture supporting VT-d, the IOMMU manages the access control policy between the hardware devices and some
+        of the system critical units such as the memory.
+
+rationale: |-
+    On x86 architectures, activating the I/OMMU prevents the system from arbritrary accesses potentially made by
+        hardware devices.
+
+severity: unknown
+
+identifiers:
+    cce@rhel7: CCE-82351-8
+    cce@rhel8: CCE-83920-9
+    cce@rhel9: CCE-83844-1
```

Comment:
How come that those CCEs have appeared here, but they are not taken from the CCE pool?

You can either remove them, or you can assign them using the `utils/fix_rules.py` script (but you may need to rebase before that).